### PR TITLE
[Ft #153727852] User help centre and live chat

### PR DESCRIPTION
**Pull request by indungu**
## PR Status
**READY**

## Story ID
[#153727852](https://www.pivotaltracker.com/story/show/153727852)

## Added Migrations
YES: Created new Models in `hc/front/models.py` for `FAQ` and `Tutorial`.

## Description
The system documentation is limited to the README and the homepage with screenshots of the dashboard.

We were tasked to meet the following acceptance criteria:
> GIVEN I am a new user to the platform and I am interested in getting onboard
> WHEN I look for the documentation on the site concerning how to start and find my way on the platform
> THEN I should find a myriad of tutorials, help videos, live chat, FAQ's etc that will ease my process of getting onboard and help answer some common questions that may arise.

## New Packages Added
None.

## Related PRs 
None

## Todos
- [x] Tested and working locally
- [x] Tested and working on development environment
- [ ] Reviewed and approved by the scrum master
- [ ] Reviewed and approved by a TTL
- [ ] Deployment to team Heroku account

## Impacted Areas in the application
1. Frontend views and templates with updated content
2. Static files for styling and chat widget integration.

## Known Issues (optional)
New users are only prompted to provide their identity (Name and email address) only when the chat agent is offline.
If the chat agent is online and the user is not registered to the site yet, an ID is autogenerated and auto assigned to the user using a combination of basic IP details.

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -170,5 +171,24 @@
     <script src="{% static 'js/bootstrap.min.js' %}"></script>
     {% endcompress %}
     {% endblock %}
+    <!-- begin olark code -->
+    <script type="text/javascript" async>
```
Comment (johnseremba) [inline templates/base.html]: Javascript code should never be in the HTML template. Create a separate `.js` file an just include it.

File: templates/base_bare.html [hunk 2]
```diff
@@ -21,5 +21,24 @@
...
+      y.identify=function(i){y("identify",k.i=i)};
+      y.configure=function(i,j){y("configure",i,j);k.c[i]=j};
+      k=y._={s:[],t:[+new Date],c:{},l:a};
+      })(window,document,"static.olark.com/jsclient/loader.js");
+      /* Add configuration calls below this comment */
+      olark.identify('7663-913-10-2758');
+      var name = "{{ request.user.email }}"
+      olark('api.chat.updateVisitorNickname', {
+  snippet: name
+  });
```
Comment (johnseremba) [inline templates/base_bare.html]: SAME AS PREVIOUS COMMENT

File: templates/front/base_docs.html [hunk 3]
```diff
@@ -22,6 +32,9 @@
         <li {% if section == "api" %} class="active" {% endif %}>
             <a href="{% url 'hc-docs-api' %}">REST API</a>
         </li>
+        <li {% if section == "faq" %} class="active" {% endif %}>
+            <a href="{% url 'hc-docs-faq' %}">FAQ</a>
```
Comment (johnseremba) [inline templates/front/base_docs.html:34]: Be consistent with the quotation marks you use. choose between double or single.

File: templates/front/docs_faq.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+{% load staticfiles %}
+
+{% block title %}FAQ - healthchecks.io{% endblock %}
+
+{% block docs_content %}
+<div class="faq">
+<h2>Frequently Asked Questions</h2>
+<h2>General</h2>
+<a name="What is Healthchecks?"></a>
+<h2 class="rule">What is Healthchecks?</h2>
```
Comment (johnseremba) [inline templates/front/docs_faq.html]: FAQs shouldn't be hardcoded.
Create another model or database table for the FAQs.
It should be possible to have them dynamically updated.

**Review by johnseremba — CHANGES_REQUESTED** [review]
Look at the comments.

File: templates/front/base_docs.html (hunk 3 above)
Comment (indungu) [reply templates/front/base_docs.html:34]: Hey @SerryJohns , I beg to differ, in this particular `line 36`, the two types of quotes are used appropriately. The `'hc-docs-faq'` are nested under the `href=""` property of the `<a>,<\a>` tag.

Comment (johnseremba) [reply templates/front/base_docs.html:34]: cool

**Review by johnseremba — APPROVED** [review]

**Pull request merged**